CHANGELOG.md

```diff
@@ -7,7 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [0.26.0] - unreleased
 
 ### Changed
+* IBAN country / address country default is Germany "DE" #APPS-589
```

Comment:
Hast du nun https://github.com/snabble/snabble-ios-sdk/pull/223 auch noch hier reingezogen und nun ist doch beides in einem?!